module-domain/src/main/kotlin/site/yourevents/invitationinformation/service/InvitationInformationService.kt

```diff
@@ -0,0 +1,33 @@
+package site.yourevents.invitationinformation.service
+
+import jakarta.transaction.Transactional
+import org.springframework.stereotype.Service
+import site.yourevents.invitation.exception.InvitationNotFoundException
+import site.yourevents.invitation.port.`in`.InvitationUseCase
+import site.yourevents.invitationinformation.domain.InvitationInformation
+import site.yourevents.invitationinformation.port.`in`.InvitationInformationUseCase
+import site.yourevents.invitationinformation.port.out.InvitationInformationPersistencePort
+import java.time.LocalDateTime
+import java.util.*
+
+@Service
+@Transactional
+class InvitationInformationService(
+    private val invitationInformationPersistencePort: InvitationInformationPersistencePort,
+    private val invitationUseCase: InvitationUseCase
+) : InvitationInformationUseCase {
+    override fun createInvitationInformation(invitationId: UUID, title: String, schedule: LocalDateTime, location: String, remark: String): InvitationInformation {
```

Comment:
매개변수가 상당하군요.. 코틀린은 횡으로 코드가 늘어나는 것을 무척? 싫어하게 때문에 개행 처리해주면 더욱 읽기 편할 것 같아요!!
```suggestion
    override fun createInvitationInformation(
        invitationId: UUID, 
        title: String, 
        schedule: LocalDateTime, 
        location: String, 
        remark: String
    ): InvitationInformation {
```

Reply:
개행 처리 진행하였습니다!!